js/index.js

```diff
@@ -40,3 +41,83 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let links = document.querySelectorAll("nav a");
+//console.log(links);
+links[0].innerHTML = siteContent['nav']['nav-item-1'];
+links[1].innerHTML = siteContent['nav']['nav-item-2'];
+links[2].innerHTML = siteContent['nav']['nav-item-3'];
+links[3].innerHTML = siteContent['nav']['nav-item-4'];
+links[4].innerHTML = siteContent['nav']['nav-item-5'];
+links[5].innerHTML = siteContent['nav']['nav-item-6'];
+
+
+
+
+links[0].style.color = "green";
+links[1].style.color = "green";
+links[2].style.color = "green";
+links[3].style.color = "green";
+links[4].style.color = "green";
+links[5].style.color = "green";
+
+
+
+
+
+
+let cta = document.querySelector('h1')
+cta.innerHTML = siteContent['cta']['h1'];
```

Comment:
Don't forget to add <br> tags or however you want to make sure that there are line breaks on the "DOM Is Awesome" text